postgres-appliance/configure_spilo.py

```diff
@@ -147,6 +147,13 @@ def deep_update(a, b):
   pg_hba:
     - hostssl all all 0.0.0.0/0 md5
     - host    all all 0.0.0.0/0 md5
+  users:
```

Comment:
Can we change it a little bit?
Entire user section should appear only if PGPASSWORD_ADMIN environment variable comes from outside.

Reply:
Yeah I can do that - good point 👍